css/main.css

```diff
@@ -1,3 +1,133 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+header{
+   font-family: 'Roboto', sans-serif;
+   background-color: white;
+}
+
+body{
+   font-family: 'Roboto', sans-serif;
+   background-color: rgb(44, 42, 42);
+}
+.nav-bar{
+    position: fixed;
+    top: 0;
+    background-color: black;
+    width: 100%;
+    display: flex;
+    flex-direction: flex-start;
+    align-items: center;
+    z-index: 1; 
+    color: white;
+  }
+
+.logo{
+    height:35px;
+    width: 35px;
+    margin: auto;
+    float: left;
+}
+
+.nav-link {
+    margin: 30px;
+    
+}
+
+.boxColor{
+    background-color: white;
+}
+
+.div2{
+    width: 100%;
+    height: 400px;
+    display: flex;
+    user-select: center;
+    align-items: center;
+    background-image: url("../imges/martin-katler-HybjMHIDlUU-unsplash.jpg");
+    background-size: cover;
+    flex-direction: column;
+}
+
+.introduction{ 
+    top: 40%;
+    left: 36%;
+    font-weight: 100;
+    font-size: 20px;
+    display: flex;
+    color: cornsilk;
+}
+
+.intro-subheading{
+    display: flex;
+    padding:10px 10px; 
+    top: 65%;
+    left: 45%; 
+    font-size: 45px;
+    color: cornsilk;
+}
+
+.learn{
+    background-color: whitesmoke;
+    border: none;
+    padding:10px 10px; 
+    display: flex;
+    top: 65%
+    left: 49%; 
+    font-size: 20px;
+    color: black; 
+}
+
+.learn:hover{
+    background-color: gray);
+}
+
+.carRacer{
+    height:700px;
+    width: 670px;
+    margin: auto;
+    float: right; 
```

Comment:
Avoid the `float` rule, it's old and unused. Flexbox should be your go-to for any positioning of this kind 🙂

(in this case, if you follow my comments above about image sizing then you shouldn't need anymore positioning on the image itself).